Packs/ShiftManagement/Scripts/GetShiftsPerUser/GetShiftsPerUser.py

```diff
@@ -0,0 +1,55 @@
+import operator
+from functools import reduce
+from typing import Dict, List
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+
+def main():
+    user_id = demisto.args().get('userId', False)
+    if not user_id:
+        get_users_response: List = demisto.executeCommand("getUsers",
+                                                          {"current": True})
+        if is_error(get_users_response):
+            return_error(
+                f'Failed to get users: {str(get_error(get_users_response))}')
+        contents = get_users_response[0]
+        if contents and len(contents.get("Contents")) == 1:
+            user_id = contents.get("Contents")[0].get("id")
+        else:
+            return_error(f'Failed to get users: User object is empty')
+
+    get_roles_response: List = demisto.executeCommand('getRoles', {})
+    if is_error(get_roles_response):
+        return_error(f'Failed to get roles: {str(get_error(get_roles_response))}')
+
+    shifts_per_user: Dict[str, int] = {}
+    get_users_response: List = demisto.executeCommand('getUsers', {})
+    if is_error(get_users_response):
+        return_error(f'Failed to get users: {str(get_error(get_users_response))}')
```

Comment:
why do you need this again?

Reply:
The first time it checked only if ` if not user_id:`. So this is a new un related check in some cases.